custom_components/unifiprotect/sensor.py

```diff
@@ -108,6 +115,58 @@ class ProtectSensorEntityDescription(ProtectRequiredKeysMixin, SensorEntityDescr
         entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
         ufp_value="recording_settings.mode",
     ),
+    ProtectSensorEntityDescription(
+        key="stats_rx",
+        name="Received Data",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_registry_enabled_default=False,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ufp_value="stats.rx_bytes",
+    ),
+    ProtectSensorEntityDescription(
+        key="stats_tx",
+        name="Transferred Data",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_registry_enabled_default=False,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ufp_value="stats.tx_bytes",
+    ),
+    ProtectSensorEntityDescription(
+        key="oldest_recording",
+        name="Oldest Recording",
+        entity_registry_enabled_default=False,
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ufp_value="stats.video.recording_start",
+    ),
+    ProtectSensorEntityDescription(
+        key="storage_used",
+        name="Storage Used",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_registry_enabled_default=False,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ufp_value="stats.storage.used",
+    ),
+    ProtectSensorEntityDescription(
+        key="write_rate",
+        name="Disk Write Rate",
+        native_unit_of_measurement=DATA_RATE_BYTES_PER_SECOND,
+        entity_registry_enabled_default=False,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ufp_value="stats.storage.rate",
+        precision=2,
+    ),
+    ProtectSensorEntityDescription(
+        key="voltage",
+        name="Voltage",
+        device_class=DEVICE_CLASS_VOLTAGE,
+        entity_registry_enabled_default=False,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ufp_value="voltage",
+        # voltage will be null if device is not camera or not on 1.20.1+
```

Comment:
Can we avoid adding the sensor if its not supported?

Reply:
Maybe a min version required field?

Reply:
That is what the `ufp_required_field` will do. `voltage` is `float | None`. If it evals to `None` (or 0.0) it will not be added.

Reply:
I meant for that comment to be "why am I using `voltage` instead of `feature_flags.has_chime`.